packages/create-react-admin/src/cli.tsx

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+import React from 'react';
+import { render } from 'ink';
+import meow from 'meow';
+import App from './app.js';
+
+const cli = meow(
+    `
```

Comment:
I see this help when using `--help`, but not `-h`. Is it possible?

Reply:
Done